force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -250,13 +357,20 @@ public with sharing class ClassesObjectsMethods {
      * For example, createAndGetBookDetails() should return a string like "Title: A Brief History of Time, Author: Stephen Hawking".
      * @return A string containing the details of the book.
      */
-    public static String createAndGetBookDetails() {
-        // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+    
+    
+     public static String createAndGetBookDetails() {
 
+        // Create the Book class with title, author, and pages properties
+        
+        // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        Book myBook = new Book('A Brief History of Time', 'Stephen Hawking', 256);
```

Comment:
This goes back to your constructor above -- this comment is only asking for the title and author, but you've added book pages in order for it to work.